ccd-definition/CaseEventToFields/CareSupervision/manageHearings/enterHearingDetails.json

```diff
@@ -30,11 +30,10 @@
     "CaseEventID": "manageHearings",
     "CaseFieldID": "hearingTypeDetails",
     "PageFieldDisplayOrder": 3,
-    "DisplayContext": "MANDATORY",
+    "DisplayContext": "Optional",
```

Comment:
I think this needs to be `"DisplayContext": "OPTIONAL"`

Reply:
will update

Reply:
done